frontend/src/widgets/HeroWidget.astro

```diff
@@ -216,7 +216,7 @@ const subtitleClass = getTextClasses('5', mainContent?.subtitleColor);
               {mainContent.pretitle}
             </p>
           )}
-          <h1 class={`title mb-4 is-size-2-mobile is-size-2-tablet is-size-1-desktop ${getBulmaColorClass(mainContent?.titleColor, 'text')}`}>{mainContent.title}</h1>
```

Comment:
In this config (the split layout) it is nice to step the type size down a smidge at smaller breakpoints